app/src/org/commcare/utils/FormUploadUtil.java

```diff
@@ -399,6 +406,16 @@ private static String getFileContentType(File f) {
                 file,
                 FormUploadUtil.getDecryptCipher(key));
 
+        if (isRunningTest()) {
```

Comment:
I don't like that we are diverging between test and live code here. This can potentially result into an issue being present in live environment that would not be caught with these tests.